integration-tests/chopsticks/src/network/assetHub.ts

```diff
@@ -0,0 +1,65 @@
+import { setupContext, SetupOption } from '@acala-network/chopsticks-testing'
+
+import type { Config } from './types.js'
+import { initialBalanceDOT, toNumber } from '../utils.js'
+
+/// Options used to create the Spiritnet context
+export const getSetupOptions = ({
+	blockNumber = undefined,
+	wasmOverride = undefined,
+}: {
+	blockNumber?: number
+	wasmOverride?: string
+}) =>
+	({
+		endpoint: process.env.ASSETHUB_WSS || 'wss://asset-hub-polkadot-rpc.dwellir.com',
+		db: './db/assethub.db.sqlite',
+		port: toNumber(process.env.ASSETHUB_PORT) || 9003,
+		wasmOverride,
+		blockNumber,
+	}) as SetupOption
+
+export function assignKSMtoAccounts(addr: string[], balance: bigint = initialBalanceDOT) {
+	return {
+		foreignAssets: {
+			account: addr.map((addr) => [
+				[KSMAssetLocation, addr],
+				{
+					balance: balance,
+					status: 'Liquid',
+					reason: 'Consumer',
+					extra: null,
+				},
+			]),
+		},
+	}
+}
+
+/// AssetHub has no own coin. Teleported dots are used as the native token.
+export function assignDotTokensToAccounts(addr: string[], balance: bigint = initialBalanceDOT) {
+	return {
+		System: {
+			Account: addr.map((address) => [[address], { providers: 1, data: { free: balance.toString() } }]),
+		},
+	}
+}
+
+/// AssetHub ParaId
+export const paraId = 1000
+
+export const KSMAssetLocation = {
+	parents: 2,
+	interior: {
+		X1: {
+			GlobalConsensus: 'Kusama',
+		},
+	},
+}
+
+// Sibling Sovereign Account
+export const siblingSovereignAccount = '4qXPdpimHh8TR24RSk994yVzxx4TLfvKj5i1qH5puvWmfAqy'
```

Comment:
Sibling for who? Is it this chain on siblings?

Reply:
This is the sibling sovereign account of AH in other chains. The account is always the same, only the encoding is different. So instead of maintaining the account in every network file, I decided to put them only on the chain itself. It seems the naming might be confusing. Do you have a better naming?

Reply:
Yeah I would only rename. Mhh, what about `sovereignAccountOnSiblingChains`?

Reply:
Updated in: [1d88f75](https://github.com/KILTprotocol/kilt-node/pull/668/commits/1d88f7573a227429390206ac451874a67715387e)